### PR TITLE
fixed the logo disappearing issue in smaller screen sizes and adds the hover on nav-link

### PR DESCRIPTION
**Pull request by dev-abhishekkr**
This PR fixes the logo disappearing issue in small screen sizes and it
This PR adds the dynamic hover on navigation link.

Earlier the logo used to get disappear in small screen sizes.

Fixes : #193 

Screenshot (126)
Screenshot (127)

### CODE REVIEW

**Review by keshav39 — COMMENTED** [review]
this seems ok but what is the hover effect that u added? the hover effect was already there andin mobile view there no hover option

**Comment by pradeeptosarkar** [comment]
@dev-abhishekkr the build checks are failing, kindly have a look

**Review by pradeeptosarkar — CHANGES_REQUESTED** [review]
@dev-abhishekkr  the build checks are failing, kindly have a look

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -122,9 +126,12 @@ function Navbar() {
     <NavbarContainer extendNavbar={extendNavbar}>
       <NavbarInnerContainer>
         <LeftContainer>
-          <NavbarLink to = "/"><Logo src="./img/logo2.png"></Logo></NavbarLink>
```
Comment (keshav39) [inline src/components/Navbar/Navbar.jsx]: undo to this point and add the required class to this navbarlink @dev-abhishekkr

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -122,9 +126,12 @@ function Navbar() {
...
+          <Logo src="./img/logo2.png"></Logo>
```
Comment (keshav39) [inline src/components/Navbar/Navbar.jsx:151]: remove these change from the code as they are not necessary

File: src/components/Navbar/Navbar.jsx [hunk 3]
```diff
@@ -122,9 +126,12 @@ function Navbar() {
...
+          {/* <NavbarLink to="/">
+            <Logo src="./img/logo2.png"></Logo>
+          </NavbarLink> */}
           {/* <NavbarLink to="/">NSCC</NavbarLink> */}
-          <NavLinks>
+          <NavLinks className="nav-links">
```
Comment (keshav39) [inline src/components/Navbar/Navbar.jsx:156]: the check is failing because u have not closed the navlinks here

**Review by keshav39 — CHANGES_REQUESTED** [review]
follow these reviews to have the pr accepted

**Pull request closed without merging**